modules/nf-core/fcs/fcsadaptor/main.nf

```diff
@@ -28,15 +28,16 @@ process FCS_FCSADAPTOR {
     }
     def args = task.ext.args ?: '--prok' // --prok || --euk
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def FCSADAPTOR_VERSION = '0.2.3' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    def FCSADAPTOR_VERSION = '0.4.0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
     """
     /app/fcs/bin/av_screen_x \\
         -o output/ \\
         $args \\
         $assembly
 
     # compress and/or rename files with prefix
-    gzip -cf output/cleaned_sequences/* > "${prefix}.cleaned_sequences.fa.gz"
+    find output/cleaned_sequences/  -type f ! -name "*.gz" -exec gzip {} \;
```

Comment:
Just this small one as I noticed that otherwise the entire line is missing from `.command.sh` !
```suggestion
    find output/cleaned_sequences/  -type f ! -name "*.gz" -exec gzip {} \\;
```